rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -26,6 +26,4 @@
     name: "{{ item }}"
     state: present
     extra_args: "{{ pip_install_options | default('') }}"
-  with_flattened:
-    - maas_pip_packages
-    - maas_pip_dependencies
+  with_items: maas_pip_packages + maas_pip_dependencies
```

Comment:
So while the documentation says I'm right, this is actually a bug in ansible: https://github.com/ansible/ansible/issues/12249 It silently looks up strings in `with_flattened` but not `with_items` and will be fixed in a release "Soon" (post-2.0).